packages/embed/src/initializers/initialize-popovers.ts

```diff
@@ -21,6 +21,8 @@ export const initializePopovers = () => {
       buildOptionsFromAttributes(button, {
         buttonColor: 'string',
         customIcon: 'string',
+        width: 'integer',
```

Comment:
I think we should get rid of `additionalAttributes` from `buildOptionsFromAttributes` and just build all attributes for all embed types. I think there is no harm building unused options and it will make the code easier to read. WDYT?

Reply:
done!